scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_wgs.json

```diff
@@ -1,37 +0,0 @@
-{
-  "case_gatk_acnv_workflow.grouping": "SAMPLE",
-  "case_gatk_acnv_workflow.common_snp_list": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/common_snps_sample-chr20.interval_list",
-  "case_gatk_acnv_workflow.ref_fasta_fai": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.fasta.fai",
-  "case_gatk_acnv_workflow.target_file": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/ice_targets_sample-chr20.tsv",
-  "case_gatk_acnv_workflow.gatk_jar": "/home/travis/build/broadinstitute/gatk-protected/build/libs/gatk-protected.jar",
-  "case_gatk_acnv_workflow.disable_all_read_filters": "false",
-  "case_gatk_acnv_workflow.calculate_target_coverage_memory": "4",
-  "case_gatk_acnv_workflow.whole_genome_coverage_memory": "4",
-  "case_gatk_acnv_workflow.ref_fasta_dict": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.dict",
-  "case_gatk_acnv_workflow.input_bam_list": "/home/travis/build/broadinstitute/gatk-protected/scripts/acnv_cromwell_tests/case_input_wgs.tsv",
-  "case_gatk_acnv_workflow.keep_duplicate_reads": "true",
-  "case_gatk_acnv_workflow.PoN": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/wgs_test.pon",
-  "case_gatk_acnv_workflow.transform": "PCOV",
-  "case_gatk_acnv_workflow.ref_fasta": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.fasta",
-  "case_gatk_acnv_workflow.normalize_somatic_read_count_memory": "4",
-  "case_gatk_acnv_workflow.PadTargets.padding": "250",
-  "case_gatk_acnv_workflow.disable_sequence_dictionary_validation": "true",
-  "case_gatk_acnv_workflow.plots_dir": "/tmp/plots/",
-  "case_gatk_acnv_workflow.conversion_dir": "/tmp/conversions/",
-  "case_gatk_acnv_workflow.enable_gc_correction": "true",
-  "case_gatk_acnv_workflow.isWGS": "true",
-  "case_gatk_acnv_workflow.wgsBinSize": "10000",
-  "case_gatk_acnv_workflow.seg_param_alpha": "0.01",
-  "case_gatk_acnv_workflow.seg_param_nperm": "10000",
-  "case_gatk_acnv_workflow.seg_param_pmethod": "HYBRID",
-  "case_gatk_acnv_workflow.seg_param_minWidth": "2",
-  "case_gatk_acnv_workflow.seg_param_kmax": "25",
-  "case_gatk_acnv_workflow.seg_param_nmin": "200",
-  "case_gatk_acnv_workflow.seg_param_eta": "0.05",
-  "case_gatk_acnv_workflow.seg_param_trim": "0.025",
-  "case_gatk_acnv_workflow.seg_param_undoSplits": "NONE",
-  "case_gatk_acnv_workflow.seg_param_undoPrune": "0.05",
```

Comment:
Yup, the old defaults are used if not specified by setting the task-level parameters in the json.  See the command block of the PerformSegmentation task in cnv_somatic_copy_ratio_bam_workflow.wdl.